c/parallel/src/reduce.cu

```diff
@@ -206,21 +216,17 @@ struct __align__({2}) storage_t {{
 {3}
 {4}
 {5}
-using device_reduce_policy = {6}::MaxPolicy;
-
-#include <cub/detail/ptx-json/json.h>
-__device__ consteval auto& policy_generator() {{
-  return ptx_json::id<ptx_json::string("device_reduce_policy")>()
-    = cub::detail::reduce::ReducePolicyWrapper<device_reduce_policy::ActivePolicy>::EncodedPolicy();
-}};
+using device_reduce_policy = {6};
+static_assert(device_reduce_policy()(__CUDA_ARCH__) == {7}, "Host generated and JIT compiled policy mismatch");
```

Comment:
@griwes I added this check here to verify that the serialized tuning from the host (`{7}`) matches the tuning selected in device code. It works, really neat! Thx for the suggestion!